src/metrics/emd/config.vsh.yaml

```diff
@@ -14,52 +14,113 @@ info:
   metrics:
       # A unique identifier for your metric (required).
       # Can contain only lowercase letters or underscores.
-    - name: emd_mean
+    - name: emd_mean_ct
       # A relatively short label, used when rendering visualisarions (required)
-      label: EMD Mean
+      label: EMD Mean CT
```

Comment:
@LuLeom I expanded the return values to 4. So now we have mean and max calculated across all cell types, then mean and max across donors (see EMD Mean DN). See comment on the implementation in script.py file. I'll explain there.